packages/hash/frontend/src/components/Dropdowns/PageTransferDropdown.tsx

```diff
@@ -1,11 +1,54 @@
-import { useEffect, useState, VoidFunctionComponent } from "react";
+import {
+  ChangeEvent,
+  useEffect,
+  useState,
+  VFC,
+  VoidFunctionComponent,
+} from "react";
 import { useRouter } from "next/router";
-import { useMutation } from "@apollo/client";
+import { useMutation, useQuery } from "@apollo/client";
 
-import { AccountSelect } from "../layout/PageSidebar/AccountSelect";
-import { MutationTransferEntityArgs } from "../../graphql/apiTypes.gen";
+import { Box } from "@mui/material";
+import {
+  GetAccountsQuery,
+  MutationTransferEntityArgs,
+} from "../../graphql/apiTypes.gen";
 import { transferEntityMutation } from "../../graphql/queries/entityType.queries";
-import { getAccountPages } from "../../graphql/queries/account.queries";
+import {
+  getAccountPages,
+  getAccounts,
+} from "../../graphql/queries/account.queries";
+
+type AccountSelectProps = {
+  onChange: (account: string) => void;
+  value: string;
+};
+
+export const AccountSelect: VFC<AccountSelectProps> = ({ onChange, value }) => {
+  const { data } = useQuery<GetAccountsQuery>(getAccounts);
+
+  return (
+    <Box
+      component="select"
+      sx={{
```

Comment:
moved this component from `page-sidebar/` folder since it is only used in this component at the moment and we might end up removing it with the new designs